libs/Transisthor/TransisthorECSLogic/Both/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+add_subdirectory(Components)
+
+set(SRC_TEL_BOTH ${SRC_TEL_BOTH_COMP} PARENT_SCOPE)
```

Comment:
See with @LucasTesnier if there is a better way to compile, Parent_Scope is...

Reply:
It's the proper way to do this, the other methods is to create a lot of other library or to add all the file path in the head Cmakefile. So, parent scope is the most pratical way